packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5245,7 +5263,7 @@ export default class Meeting extends StatelessWebexPlugin {
       // @ts-ignore
       this.webex.internal.newMetrics.submitClientEvent({
         name: 'client.media.rx.start',
-        payload: {mediaType: data.type},
+        payload: {mediaType: data.type, shareInstanceId: this.shareInstanceId},
```

Comment:
we shouldn't be sending the event with shareInstanceId for main audio and main video, only for share, same for "client.media.rx.stop"